schema.ts

```diff
@@ -31,13 +31,19 @@ export interface ModuleDoc {
   exports?: Array<ExportDoc>;
 }
 
-export type ExportDoc = ClassDoc|FunctionDoc|VariableDoc;
+export type ExportDoc = ClassDoc | FunctionDoc | VariableDoc | DefinitionDoc;
```

Comment:
Note: I think we'll need to make a new interface for exports, but we'll do that separately.